vuepress/docs/next/docs/compose/preinstalled-widgets.md

```diff
@@ -5,37 +5,40 @@ sidebarDepth: 2
 # Preinstalled Widgets
 
 
-The Entando Platform includes a number of useful components to accelerate application development. These consist of widgets, page templates, content templates and content types. This page introduces the CMS, navigation, page, SEO and system widgets that are available out of the box. 
+The Entando Platform includes a number of useful components to accelerate application development. These consist of widgets, page templates, content templates, and content types. 
+
+This page introduces the widgets that are available out of the box with an Entando 7 installation. These widgets are categorized by type, implementing CMS, navigation, page, SEO or system functionality.
```

Comment:
'implementing' is confusing and awkward. After categorized by type, you're expecting nouns, not a verb so it reads strangely. Like is 'implementing cms' a type

Reply:
it would need to be "type:" to read that way. but i can see that right now after "type," you're expecting additional details about widgets instead of about widget types, like "the widgets are categorized by type, use x, look like y..." like more verbs of the same tense to describe additional characteristics.